src/views/projects.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+
+    <meta charset="en">
+
+<head>
```

Comment:
You already have a `<head>` tag on line 4, is this tag meant to close? You also need a `<body>` tag for the other content on the page.

```suggestion
</head>
<body>
```